### DISCUSSION TITLE
Recognize musical notes and play them

### DISCUSSION THREAD

**Discussion by francoisschwarzentruber** [tableaunoir/tableaunoir#130] [Ideas]
![image](https://user-images.githubusercontent.com/43071857/103103896-82134500-4624-11eb-9fec-771fc81d5a14.png)

**Comment by Naereen** [comment]
I'm not a music expert, but I honestly think that this would be hard to implement correctly...

**Comment by francoisschwarzentruber** [comment]
Probably... and it may be the case that some note is not detected... or a note is detected but erased and Tableaunoir may think it is still there...
Also, not very it will be useful.
